src/core/client/stream/tabs/Profile/Settings/DeleteAccount/Pages/Common/PageStepBar.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FunctionComponent } from "react";
+
+import { Step, StepBar } from "coral-ui/components";
+
+import styles from "./PageStepBar.css";
+
+interface Props {
+  step: number;
+}
+
+const PageStepBar: FunctionComponent<Props> = ({ step }) => {
+  return (
+    <div>
+      <StepBar currentStep={step} className={styles.stepBar}>
+        <Step lineClassName={styles.line}>{""}</Step>
```

Comment:
Check with @IAmSamHankins around why this is different than the install wizard component that uses actual string contents inside the step components instead of `{""}`.